net/quagga/src/opnsense/scripts/quagga/diag-bgp.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+case "$1" in
+  bgp)
+    vtysh -d bgpd -c "show ip bgp"
+    ;;
+  summary)
+    vtysh -d bgpd -c "show ip bgp summary"
+    ;;
+  neighbor)
+    vtysh -d bgpd -c "show ip bgp neighbors $2"
+    ;;
+  neighbor-adv)
+    vtysh -d bgpd -c "show ip bgp neighbors $2 advertised-routes"
+    ;;
+  *)
+    echo "Usage: $0 bgp|summary|neighbor <ip>|neighbor-adv <ip>"
+    exit 1
+esac
+exit 0
```

Comment:
I don't think this will parse the output so it is not good for the API. I would use a script to create a JSON object from the raw output. If you paste an output for all of them, I can write one in Ruby (in the plugins I am allowed to use it :) )